lib/msal-node/docs/request.md

```diff
@@ -0,0 +1,38 @@
+# Request
+
+Since MSAL Node supports various authorization code grants, there is support for different public APIs per grant and the corresponding request.
+
+## Authorization Code Flow
+
+### Public APIs
+- [getAuthCodeUrl()](https://azuread.github.io/microsoft-authentication-library-for-js/ref/msal-node/classes/_src_client_publicclientapplication_.publicclientapplication.html#getauthcodeurl): This API is the first leg of the `authorization code grant` for MSAL Node. The request is of the type [AuthorizationUrlRequest](https://azuread.github.io/microsoft-authentication-library-for-js/ref/msal-common/modules/_src_request_authorizationurlrequest_.html).
+The application is sent a URL that can be used to generate an `authorization code` by any browser of choice, and then proceed to call the below API for the `token acquisition` .
```

Comment:
"can be opened in a browser of choice, where the user can input their credential, and will be redirected back to the redirect URI registered with an authorization code. The authorization code can be redeemed for a token..."

Reply:
Done.